common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -56,6 +56,7 @@
     //http://jmenv.tbsite.net:8080/rocketmq/nsaddr
     //public static final String WS_ADDR = "http://" + WS_DOMAIN_NAME + ":8080/rocketmq/" + WS_DOMAIN_SUBGROUP;
     public static final String AUTO_CREATE_TOPIC_KEY_TOPIC = "TBW102"; // Will be created at broker when isAutoCreateTopicEnable
+    public static final String SCHEDULE_TOPIC = "SCHEDULE_TOPIC_XXXX";
```

Comment:
MixAll class should not inflate continue. could you help to move the system topic related logic to a new class, such as TopicValidator or TopicChecker?

Reply:
It's a good way to make code cleaner.I'will try my best to do it